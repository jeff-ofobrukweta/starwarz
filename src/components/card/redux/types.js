export const SET_ACTIVE = "SET_ACTIVE";
export const ADDED_TO_FAVOURITES = "ADDED TO FAVOURITES";
