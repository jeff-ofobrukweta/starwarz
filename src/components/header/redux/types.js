export const SET_NAV = "SET_NAV";
