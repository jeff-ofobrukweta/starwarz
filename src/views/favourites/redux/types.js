export const GET_FAVOURITES = "GET_FAVOURITES";
