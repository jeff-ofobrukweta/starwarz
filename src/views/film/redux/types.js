export const SET_FILM = "SET_FILM";
