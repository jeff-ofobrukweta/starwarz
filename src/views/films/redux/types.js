export const SET_FILMS = "SET_FILMS";
