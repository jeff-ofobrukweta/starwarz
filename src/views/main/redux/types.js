const GET_FILM = "GET_FILM";
const SET_FILM = "SET_FILM";
