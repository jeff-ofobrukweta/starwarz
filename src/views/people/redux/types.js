export const SET_PEOPLE = "SET_PEOPLE";
