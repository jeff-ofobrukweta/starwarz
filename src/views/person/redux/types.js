export const SET_PERSON = "SET_PERSON";
