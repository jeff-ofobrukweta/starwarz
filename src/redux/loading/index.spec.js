// ./**tests**/loading.js
import configureMockStore from 'redux-mock-store'
import { LOADING } from './types';

// In a real application we would import this from our actions
const setLoading = (payload) => ({
  type: LOADING,
  payload
});

// Create a mock store
const mockStore = configureMockStore()
const store = mockStore({})
describe('action creators', () => {
  it('creates LOADING when loading was successful', () => {
    // Dispatch the setLoading action with the values of a new to-do.
    store.dispatch(setLoading(false));
    expect(store.getActions()).toMatchSnapshot();
  });
});