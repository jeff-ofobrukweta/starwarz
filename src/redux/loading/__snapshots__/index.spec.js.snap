// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action creators creates CREATE_SUCCESS when creating a to-do was successful 1`] = `
Array [
  Object {
    "payload": false,
    "type": "LOADING",
  },
]
`;

exports[`action creators creates LOADING when loading was successful 1`] = `
Array [
  Object {
    "payload": false,
    "type": "LOADING",
  },
]
`;
